### ISSUE TITLE
Broken dependency for faker

### ISSUE THREAD

**Issue by jdblack** [blocoio/faker#31]
I came across the following while trying to use  this faker 2.0.3  in a  newish (API 34)  kotlin android app that I'm writing

11: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:checkDebugAndroidTestDuplicateClasses'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.github.bmoliveira:snake-yaml:v1.18-android.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/github/bmoliveira/snake-yaml/v1.18-android/snake-yaml-v1.18-android.pom
       - https://repo.maven.apache.org/maven2/com/github/bmoliveira/snake-yaml/v1.18-android/snake-yaml-v1.18-android.pom
     Required by:
         project :app > io.bloco:faker:2.0.3

**Comment by jdblack** [comment]
To fix this, one needs to add  jitpack to settings.gradle.kts

```
pluginManagement {
    repositories {
        google()
        mavenCentral()
        gradlePluginPortal()
    }
}
dependencyResolutionManagement {
    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
    repositories {
        google()
        mavenCentral()
        maven("https://jitpack.io")  <-------------------
    }
}
```

**Comment by sdsantos** [comment]
Fixed in version 2.0.4
